cmd/olympus/main.go

```diff
@@ -76,6 +81,6 @@ func getWorker() worker.Worker {
 			},
 		},
 		Name:           actions.OlympusWorkerName,
-		MaxConcurrency: 25,
+		MaxConcurrency: 5,
```

Comment:
Is this maximum Goroutines or Redis Connections? If Goroutines, then it's kinda small no? 
Also the other `getWorker` function for the proxy still uses `25` :)

Reply:
yes i want to use number of goroutines, but wnated to share this before i go offline, i will continue on this tomorrow